src/dotnet-roslyn-tools/Commands/CreateReleaseTagsCommand.cs

```diff
@@ -13,10 +14,15 @@ internal class CreateReleaseTagsCommand
 {
     private static readonly CreateReleaseTagsCommandDefaultHandler s_defaultHandler = new();
 
+    private static readonly string[] s_allProductNames = VSBranchInfo.AllProducts.Select(p => p.Name.ToLower()).ToArray();
+
+    internal static readonly Option<string> ProductOption = new Option<string>(new[] { "--product", "-p" }, () => "roslyn", "Which product to get info for (roslyn or razor)").FromAmong(s_allProductNames);
```

Comment:
So it looks like s_allProductNames will also include TypeScript and F# -- would those be expected to work or not? The hint text is unclear.

Reply:
I would expect it to work for anything we have a `Product` class for, as thats kinda the whole point of the abstraction. I probably copied this message from `vsbranchinfo`, and that was probably written before F# and TypeScript were added.